src/modules/developer-guide/pages/web-applications/index.adoc

```diff
@@ -34,7 +34,18 @@ Core contributors should skip this part.
 === Configure the environment
 
 .Inside your web application directory:
+
+For Linux:
+
 . run `cp public/config.example.js public/config.js`
+
+For Windows:
+
+. in `public` folder run `copy config.example.js config.js`
+
+After that:
+
+[start=2]
```

Comment:
Can you place both of these options into a single bullet, and remove the "After that: ..." part; for example:

> Run `cp public/config.example.js public/config.js` for *Linux*, or `copy config.example.js config.js` for *Windows*

We will need to do something more propper later on (for different platforms), but for now it will do.

Reply:
okay, no problem